ceph_salt/config_shell.py

```diff
@@ -345,6 +345,21 @@ def possible_values(self):
                     },
                 }
             },
+            'Registries': {
```

Comment:
nit: Since https://github.com/ceph/ceph-salt/pull/166 we are now using lowercase on config nodes, can you please rename it?

Reply:
Thanks for reminding me. Fixed too.